cookbook-react-native/recipes/buttons/custom-hightlight-button/constants/platform.ts

```diff
@@ -0,0 +1,5 @@
+import { Dimensions } from 'react-native';
```

Comment:
We'll need to find a way of avoiding the repetition of this constants files.
Maybe creating a package for all common utils used in cookbook recipes?